config.yml

```diff
@@ -38,6 +39,7 @@ default:
   indices_timestamp: "20211231"
   market_share_target_reference_year: 2021
   time_horizon: 5
+  tdm_delta_years: [5, 10]
```

Comment:
I think maybe it is conceptually better to allow the `2021Q4` and `2022Q4` configs to inherit this parameter from `default`, because otherwise it implies that this value is specific to this config

(any parameters in `default` but not in another named config are automatically inherited, e.g. `update_factset`)

Reply:
Ok, I was just following suit to other parameters (for example `time_horizon` is also likely to be identical between 2021Q4 and 2022Q4 runs)

Reply:
#36

Reply:
yes, I believe any values found in `default` that are not found in another config are inherited, e.g. `update_factset`

that's why this works when the `2021Q4` config is used
https://github.com/RMI-PACTA/workflow.data.preparation/blob/cf842c281b00df695f5a5433cb6ffed5cecab330/run_pacta_data_preparation.R#L24-L40

---

https://rstudio.github.io/config/articles/inheritance.html
> Other configurations automatically inherit all default values so need only define values specialized for that configuration.